src/cli/cli_config.rs

```diff
@@ -137,12 +137,12 @@ pub struct DependencyConfig {
     pub pypi: bool,
 
     /// The platform(s) for which the dependency should be modified
-    #[arg(long, short)]
-    pub platform: Vec<Platform>,
+    #[arg(long = "platform", short)]
+    pub platforms: Vec<Platform>,
 
     /// The feature for which the dependency should be modified
-    #[arg(long, short)]
-    pub feature: Option<String>,
+    #[clap(long, short, default_value_t)]
+    pub feature: FeatureName,
```

Comment:
Not sure but I believe we used a later parsing logic for it to give a better "miette" error.

Reply:
It doesn't. Creating a `FeatureName` always works.